lib/configure.js

```diff
@@ -279,32 +303,75 @@ module.exports = function() {
 
       // We don't need to save the Travis token, because the Travis module
       // caches it in the Travis instance.
+
+      cli.spinner(chalk.green('✓ ') + message + ' done!', true);
     });
   })
 
   .then(function() {
     // Now that we have the Travis token, delete the temporary GitHub token.
 
-    process.stdout.write('Deleting temporary GitHub token for getting Travis token…\n');
+    var message = 'Deleting temporary GitHub token for getting Travis token…';
+    cli.spinner('  ' + message);
+
+    return deleteToken(tempTokenId)
+    .then(function() {
+      cli.spinner(chalk.green('✓ ') + message + ' done!', true);
+    });
+  })
 
-    return deleteToken(tempTokenId);
+  .then(function() {
+    var message = 'Creating permanent GitHub token for Travis to push to the repository…';
+    spinnerMessage = message;
+    cli.spinner('  ' + spinnerMessage);
+
+    return createToken(['public_repo'], 'Oghliner token for ' + slug, noteUrl)
+    .then(function(res) {
+      token = res.token;
+      cli.spinner(chalk.green('✓ ') + message + ' done!', true);
+      spinnerMessage = null;
+    });
   })
 
   .then(function() {
     function ensureActiveInTravis() {
-      process.stdout.write('Checking the status of your repository in Travis…\n');
+      var message = 'Checking the status of your repository in Travis…';
+      cli.spinner('  ' + message);
       return travis.hooks.get()
       .then(function(res) {
+        cli.spinner(chalk.green('✓ ') + message + ' done!', true);
+        return res.hooks;
+      })
+      .then(function(hooks) {
         var hook;
-        for (var i = 0; i < res.hooks.length; i++) {
-          hook = res.hooks[i];
+        for (var i = 0; i < hooks.length; i++) {
+          hook = hooks[i];
           if (hook.owner_name === user && hook.name === repo) {
             if (hook.active) {
-              process.stdout.write('Good news, your repository is already active in Travis!\n');
+              process.stdout.write(
+                '\n' +
+                'Good news, your repository is already active in Travis!\n' +
+                '\n'
+              );
               return;
             }
-            process.stdout.write('Your repository isn\'t active in Travis yet.  Activating it…\n');
-            return promisify(travis.hooks(hook.id).put)({ hook: { active: true } });
+
+            var message = 'Your repository isn\'t active in Travis yet; activating it…';
+            cli.spinner('  ' + message);
+            return promisify(travis.hooks(hook.id).put)({ hook: { active: true } })
+            .then(function(res) {
+              if (res.result) {
+                cli.spinner(chalk.green('✓ ') + message + ' done!', true);
+              } else {
+                cli.spinner(chalk.red('✓ ') + message + ' failed!', true);
```

Comment:
> Maybe we can use a 'X' here instead of a 'V'?

Good idea, implemented in df62a56 (where I also change "failed!" to "error!" for consistency with *integrate*).